server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -2537,6 +2543,17 @@ async def delete_application_records(
             endpoint_ids=endpoint_ids,
         )
 
+    def check_model_monitoring_status(self) -> list[str]:
```

Comment:
```suggestion
    def get_deployed_model_monitoring_functions(self) -> list[str]:
```
You don't really check MM status here, but rather just return what is deployed.